package.json

```diff
@@ -1,55 +1,65 @@
 {
-  "name": "propguard",
-  "version": "0.1.0",
+  "name": "gatsby-starter-default",
```

Comment:
If you don't mind please change the
1. name to Propguard
2. the description to "Propguard is a webapp that teaches people how to start writing basic flight code with technologies like PX4, MAVSDK, and Python." 
3. author to Multirotor Design Team